src/rabbit_watchdog_disk.erl

```diff
@@ -23,8 +23,8 @@
 -export([init/1, validate/1, action/1, terminate/1]).
 
 -define(FORBIDDEN_UNIX_DIRS,
-   ["/usr", "/etc", "/opt", "/bin", "/home", "/opt", "/mnt", "/sbin", "/bin",
-    "/dev", "/Volumes", "/Users", "/System", "/Applications", "/Library"
+   ["/usr", "/etc", "/opt", "/bin", "/home(/\\w+)?", "/opt", "/mnt", "/sbin",
```

Comment:
Neither /home, nor /home/<user>/ can be deleted. But other subdirs may be deleted, as mentioned below.